[2025-01-01T23:26:55.564+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-01T23:26:55.592+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prediction_dag_v1.predict_model manual__2025-01-01T23:26:47.120226+00:00 [queued]>
[2025-01-01T23:26:55.608+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prediction_dag_v1.predict_model manual__2025-01-01T23:26:47.120226+00:00 [queued]>
[2025-01-01T23:26:55.610+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-01T23:26:55.633+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): predict_model> on 2025-01-01 23:26:47.120226+00:00
[2025-01-01T23:26:55.642+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=96) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-01T23:26:55.645+0000] {standard_task_runner.py:72} INFO - Started process 98 to run task
[2025-01-01T23:26:55.646+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'prediction_dag_v1', 'predict_model', 'manual__2025-01-01T23:26:47.120226+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/02_prediction/02_prediction_v1.py', '--cfg-path', '/tmp/tmpta3dedsl']
[2025-01-01T23:26:55.648+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask predict_model
[2025-01-01T23:26:55.716+0000] {task_command.py:467} INFO - Running <TaskInstance: prediction_dag_v1.predict_model manual__2025-01-01T23:26:47.120226+00:00 [running]> on host 24eea4b1b7f8
[2025-01-01T23:26:56.127+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cm' AIRFLOW_CTX_DAG_ID='prediction_dag_v1' AIRFLOW_CTX_TASK_ID='predict_model' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T23:26:47.120226+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-01T23:26:47.120226+00:00'
[2025-01-01T23:26:56.129+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-01T23:26:56.131+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-01T23:26:56.132+0000] {logging_mixin.py:190} INFO - Current task name:predict_model state:running start_date:2025-01-01 23:26:55.594352+00:00
[2025-01-01T23:26:56.132+0000] {logging_mixin.py:190} INFO - Dag name:prediction_dag_v1 and current dag run status:running
[2025-01-01T23:26:56.133+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-01T23:26:56.135+0000] {process_utils.py:186} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venv8clpe0pi --system-site-packages --python=python
[2025-01-01T23:26:56.137+0000] {process_utils.py:190} INFO - Output:
[2025-01-01T23:26:57.428+0000] {process_utils.py:194} INFO - created virtual environment CPython3.12.7.final.0-64 in 670ms
[2025-01-01T23:26:57.431+0000] {process_utils.py:194} INFO -   creator CPython3Posix(dest=/tmp/venv8clpe0pi, clear=False, no_vcs_ignore=False, global=True)
[2025-01-01T23:26:57.432+0000] {process_utils.py:194} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2025-01-01T23:26:57.433+0000] {process_utils.py:194} INFO -     added seed packages: pip==24.3.1
[2025-01-01T23:26:57.434+0000] {process_utils.py:194} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-01-01T23:26:57.461+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venv8clpe0pi/bin/pip install -r /tmp/venv8clpe0pi/requirements.txt
[2025-01-01T23:26:57.462+0000] {process_utils.py:190} INFO - Output:
[2025-01-01T23:26:59.080+0000] {process_utils.py:194} INFO - Collecting feature-engine==1.8.2 (from -r /tmp/venv8clpe0pi/requirements.txt (line 1))
[2025-01-01T23:26:59.196+0000] {process_utils.py:194} INFO -   Downloading feature_engine-1.8.2-py2.py3-none-any.whl.metadata (9.9 kB)
[2025-01-01T23:26:59.252+0000] {process_utils.py:194} INFO - Collecting joblib==1.3.2 (from -r /tmp/venv8clpe0pi/requirements.txt (line 2))
[2025-01-01T23:26:59.264+0000] {process_utils.py:194} INFO -   Downloading joblib-1.3.2-py3-none-any.whl.metadata (5.4 kB)
[2025-01-01T23:26:59.274+0000] {process_utils.py:194} INFO - Requirement already satisfied: numpy==1.26.4 in /home/***/.local/lib/python3.12/site-packages (from -r /tmp/venv8clpe0pi/requirements.txt (line 3)) (1.26.4)
[2025-01-01T23:26:59.498+0000] {process_utils.py:194} INFO - Collecting pandas==2.2.3 (from -r /tmp/venv8clpe0pi/requirements.txt (line 4))
[2025-01-01T23:26:59.508+0000] {process_utils.py:194} INFO -   Downloading pandas-2.2.3-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (89 kB)
[2025-01-01T23:26:59.575+0000] {process_utils.py:194} INFO - Requirement already satisfied: pendulum in /home/***/.local/lib/python3.12/site-packages (from -r /tmp/venv8clpe0pi/requirements.txt (line 5)) (3.0.0)
[2025-01-01T23:26:59.727+0000] {process_utils.py:194} INFO - Collecting scikit-learn==1.5.2 (from -r /tmp/venv8clpe0pi/requirements.txt (line 6))
[2025-01-01T23:26:59.749+0000] {process_utils.py:194} INFO -   Downloading scikit_learn-1.5.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (13 kB)
[2025-01-01T23:26:59.944+0000] {process_utils.py:194} INFO - Collecting scipy>=1.4.1 (from feature-engine==1.8.2->-r /tmp/venv8clpe0pi/requirements.txt (line 1))
[2025-01-01T23:26:59.953+0000] {process_utils.py:194} INFO -   Downloading scipy-1.14.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
[2025-01-01T23:27:00.096+0000] {process_utils.py:194} INFO - Collecting statsmodels>=0.11.1 (from feature-engine==1.8.2->-r /tmp/venv8clpe0pi/requirements.txt (line 1))
[2025-01-01T23:27:00.103+0000] {process_utils.py:194} INFO -   Downloading statsmodels-0.14.4-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (9.2 kB)
[2025-01-01T23:27:00.123+0000] {process_utils.py:194} INFO - Requirement already satisfied: python-dateutil>=2.8.2 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.2.3->-r /tmp/venv8clpe0pi/requirements.txt (line 4)) (2.9.0.post0)
[2025-01-01T23:27:00.124+0000] {process_utils.py:194} INFO - Requirement already satisfied: pytz>=2020.1 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.2.3->-r /tmp/venv8clpe0pi/requirements.txt (line 4)) (2024.2)
[2025-01-01T23:27:00.125+0000] {process_utils.py:194} INFO - Requirement already satisfied: tzdata>=2022.7 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.2.3->-r /tmp/venv8clpe0pi/requirements.txt (line 4)) (2024.2)
[2025-01-01T23:27:00.182+0000] {process_utils.py:194} INFO - Collecting threadpoolctl>=3.1.0 (from scikit-learn==1.5.2->-r /tmp/venv8clpe0pi/requirements.txt (line 6))
[2025-01-01T23:27:00.201+0000] {process_utils.py:194} INFO -   Downloading threadpoolctl-3.5.0-py3-none-any.whl.metadata (13 kB)
[2025-01-01T23:27:00.221+0000] {process_utils.py:194} INFO - Requirement already satisfied: time-machine>=2.6.0 in /home/***/.local/lib/python3.12/site-packages (from pendulum->-r /tmp/venv8clpe0pi/requirements.txt (line 5)) (2.16.0)
[2025-01-01T23:27:00.224+0000] {process_utils.py:194} INFO - Requirement already satisfied: six>=1.5 in /home/***/.local/lib/python3.12/site-packages (from python-dateutil>=2.8.2->pandas==2.2.3->-r /tmp/venv8clpe0pi/requirements.txt (line 4)) (1.16.0)
[2025-01-01T23:27:00.282+0000] {process_utils.py:194} INFO - Collecting patsy>=0.5.6 (from statsmodels>=0.11.1->feature-engine==1.8.2->-r /tmp/venv8clpe0pi/requirements.txt (line 1))
[2025-01-01T23:27:00.291+0000] {process_utils.py:194} INFO -   Downloading patsy-1.0.1-py2.py3-none-any.whl.metadata (3.3 kB)
[2025-01-01T23:27:00.301+0000] {process_utils.py:194} INFO - Requirement already satisfied: packaging>=21.3 in /home/***/.local/lib/python3.12/site-packages (from statsmodels>=0.11.1->feature-engine==1.8.2->-r /tmp/venv8clpe0pi/requirements.txt (line 1)) (24.1)
[2025-01-01T23:27:00.325+0000] {process_utils.py:194} INFO - Downloading feature_engine-1.8.2-py2.py3-none-any.whl (374 kB)
[2025-01-01T23:27:00.413+0000] {process_utils.py:194} INFO - Downloading joblib-1.3.2-py3-none-any.whl (302 kB)
[2025-01-01T23:27:00.485+0000] {process_utils.py:194} INFO - Downloading pandas-2.2.3-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.7 MB)
[2025-01-01T23:27:03.219+0000] {process_utils.py:194} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.7/12.7 MB 4.7 MB/s eta 0:00:00
[2025-01-01T23:27:03.243+0000] {process_utils.py:194} INFO - Downloading scikit_learn-1.5.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.9 MB)
[2025-01-01T23:27:07.760+0000] {process_utils.py:194} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.9/12.9 MB 2.8 MB/s eta 0:00:00
[2025-01-01T23:27:07.773+0000] {process_utils.py:194} INFO - Downloading scipy-1.14.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (40.8 MB)
[2025-01-01T23:27:14.979+0000] {process_utils.py:194} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.8/40.8 MB 5.7 MB/s eta 0:00:00
[2025-01-01T23:27:14.990+0000] {process_utils.py:194} INFO - Downloading statsmodels-0.14.4-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (10.7 MB)
[2025-01-01T23:27:16.749+0000] {process_utils.py:194} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.7/10.7 MB 6.2 MB/s eta 0:00:00
[2025-01-01T23:27:16.758+0000] {process_utils.py:194} INFO - Downloading threadpoolctl-3.5.0-py3-none-any.whl (18 kB)
[2025-01-01T23:27:16.779+0000] {process_utils.py:194} INFO - Downloading patsy-1.0.1-py2.py3-none-any.whl (232 kB)
[2025-01-01T23:27:17.617+0000] {process_utils.py:194} INFO - Installing collected packages: threadpoolctl, scipy, patsy, joblib, scikit-learn, pandas, statsmodels, feature-engine
[2025-01-01T23:27:24.060+0000] {process_utils.py:194} INFO -   Attempting uninstall: pandas
[2025-01-01T23:27:24.067+0000] {process_utils.py:194} INFO -     Found existing installation: pandas 2.1.4
[2025-01-01T23:27:24.068+0000] {process_utils.py:194} INFO -     Not uninstalling pandas at /home/***/.local/lib/python3.12/site-packages, outside environment /tmp/venv8clpe0pi
[2025-01-01T23:27:24.068+0000] {process_utils.py:194} INFO -     Can't uninstall 'pandas'. No files were found to uninstall.
[2025-01-01T23:27:33.048+0000] {process_utils.py:194} INFO - ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
[2025-01-01T23:27:33.050+0000] {process_utils.py:194} INFO - apache-***-providers-google 10.25.0 requires pandas<2.2,>=2.1.2; python_version >= "3.9", but you have pandas 2.2.3 which is incompatible.
[2025-01-01T23:27:33.050+0000] {process_utils.py:194} INFO - apache-***-providers-snowflake 5.8.0 requires pandas<2.2,>=2.1.2; python_version >= "3.9", but you have pandas 2.2.3 which is incompatible.
[2025-01-01T23:27:33.051+0000] {process_utils.py:194} INFO - Successfully installed feature-engine-1.8.2 joblib-1.3.2 pandas-2.2.3 patsy-1.0.1 scikit-learn-1.5.2 scipy-1.14.1 statsmodels-0.14.4 threadpoolctl-3.5.0
[2025-01-01T23:27:33.430+0000] {python.py:513} INFO - Use 'pickle' as serializer.
[2025-01-01T23:27:33.454+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venv8clpe0pi/bin/python /tmp/venv-call4fsdn59l/script.py /tmp/venv-call4fsdn59l/script.in /tmp/venv-call4fsdn59l/script.out /tmp/venv-call4fsdn59l/string_args.txt /tmp/venv-call4fsdn59l/termination.log
[2025-01-01T23:27:33.455+0000] {process_utils.py:190} INFO - Output:
[2025-01-01T23:27:35.304+0000] {process_utils.py:194} INFO - /home/***/.local/lib/python3.12/site-packages/***/configuration.py:859 FutureWarning: section/key [core/sql_alchemy_conn] has been deprecated, you should use[database/sql_alchemy_conn] instead. Please update your `conf.get*` call to use the new name
[2025-01-01T23:27:37.791+0000] {process_utils.py:194} INFO - 2025-01-01
[2025-01-01T23:27:38.729+0000] {python.py:240} INFO - Done. Returned value was: Sin predicción
[2025-01-01T23:27:38.751+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-01T23:27:38.755+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=prediction_dag_v1, task_id=predict_model, run_id=manual__2025-01-01T23:26:47.120226+00:00, execution_date=20250101T232647, start_date=20250101T232655, end_date=20250101T232738
[2025-01-01T23:27:38.769+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-01T23:27:38.773+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-01T23:27:38.783+0000] {logging_mixin.py:190} INFO - Dag name:prediction_dag_v1 queued_at:2025-01-01 23:26:47.145813+00:00
[2025-01-01T23:27:38.785+0000] {logging_mixin.py:190} INFO - Task hostname:24eea4b1b7f8 operator:_PythonVirtualenvDecoratedOperator
[2025-01-01T23:27:38.832+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-01T23:27:38.864+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-01T23:27:38.869+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
